src/modules/ekf2/EKF/ekf.h

```diff
@@ -490,8 +490,11 @@ class Ekf final : public EstimatorInterface
 
 	Dcmf _R_to_earth{};	///< transformation matrix from body frame to earth frame from last EKF prediction
 
-	Vector2f _accel_lpf_NE{};			///< Low pass filtered horizontal earth frame acceleration (m/sec**2)
-	float _height_rate_lpf{0.0f};
+	static constexpr float _accel_horiz_lpf_time_constant = 1.f;
```

Comment:
Just an observation: I use capital letters for these defines some modules contain _kxxx as naming convention and here it's not obvious that it's a constant at all... I'm not saying you have to follow my convention but generally it would be nice to make it obvious.

Reply:
You're right, the google style guide also suggests to use something like `_kConstantVariableName`; I'll change that.

Reply:
This was addressed 👍